packages/mobile/components/RecoveryPhrase.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="typescript">
+    export let recoveryPhrase = []
+    export let verifyRecoveryPhrase = undefined
+
+    export let hide = false
+    export let classes = ''
+</script>
+
+{#if recoveryPhrase}
+    <div
+        data-label="recovery-phrase"
+        class="grid w-full text-12 grid-cols-3 overflow-y-auto p-3 rounded-2xl border border-solid border-gray-300 {classes}"
+        class:hide
+    >
+        {#each recoveryPhrase as word, i}
+            <span
+                id="recovery-word-{i}"
+                class="p-3 flex flex-row items-center bg-transparent text-gray-500 border border-solid border-transparent"
+                class:errored={verifyRecoveryPhrase &&
+                    verifyRecoveryPhrase[i] &&
+                    verifyRecoveryPhrase[i] !== recoveryPhrase[i]}
+            >
+                <span class="text-gray-500 mr-2">{`${i + 1}. `}</span>
+                <span class="'text-gray-500 dark:text-white"
+                    >{verifyRecoveryPhrase && verifyRecoveryPhrase[i] !== recoveryPhrase[i] ? '*****' : word}</span
+                >
+            </span>
+        {/each}
+    </div>
+{/if}
+
+<style type="text/scss">
+    div {
+        max-width: 460px;
+        &.hide {
+            filter: blur(4px);
+        }
+        span {
+            &.errored {
```

Comment:
In figma it shows how we should always highlight the current word you are verifying, and if wrong then it red be red
![image](https://user-images.githubusercontent.com/3624944/193814012-e36a9dd5-7508-43f6-9250-c4986327ad44.png)
![image](https://user-images.githubusercontent.com/3624944/193814084-97da1e5d-08a2-42f8-afac-b1f5a2eb192b.png)

Reply:
Though we can polish this later on in a polish PR